### ISSUE TITLE
Training sample for 'checkpoints/results.csv'

### ISSUE THREAD

**Issue by ErikBag** [ChenPy00/PPB-Affinity#6]
Hello!

Could you tell me which training sample was used for your model, from which the dG_pred values in the file 'checkpoints/results.csv' were predicted?

**Comment by ErikBag** [comment]
@ChenPy00,  I urgently need your answer, pleese.

**Comment by ChenPy00** [comment]
@ErikBag , I’m so sorry I missed your message. I apologize for the oversight.
The result file is generated through [validate function](https://github.com/ChenPy00/PPB-Affinity/blob/main/train.py#L126), which means that each sample is a validation set.
I have uploaded two files containing my data partitioning information. They are located in the `train_datasplit` folder and can be read as follows:
```python
import pickle
with open('./train_datasplit/train_pdb.pkl', 'rb') as f:
	data = pickle.load(f)
```
I hope it can help you.

**Comment by ErikBag** [comment]
Thank you for your answer
Do I understand correctly that among the data on which you predicted the metrics for the article(https://www.nature.com/articles/s41597-024-03997-4), there are also complexes that were in the training sample?

**Comment by ChenPy00** [comment]
When we train the model, we adopt a ["strict"](https://github.com/ChenPy00/PPB-Affinity/blob/main/BaselineModel/datasets/mixed_dataset.py#L155) data partitioning approach, where strict refers to data partitioning based on PDB ID, where the same PDB does not appear in both the training and validation sets. 
We use cross-validation for model training, where the aggregation of the validation set for each fold is equal to the complete training set. This means that the `checkpoints/results.csv` contains the full training set, and these are all the predicted results when is used for validation.
I'm not sure I've understood your question correctly, so I hope that solves your question.
